### ISSUE TITLE
ModuleNotFoundError: No module named 'theano'

### ISSUE THREAD

**Issue by dtitov** [itdxer/neupy#198]
Hi, I've got an issue with neupy installation. 

Input: `pip install neupy`

Output:
```
Collecting neupy
  Using cached neupy-0.6.1.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/private/var/folders/ys/qjx1ds4s4jqb4sd405dw32j40000gn/T/pip-build-vjynqabb/neupy/setup.py", line 6, in <module>
        import neupy
      File "/private/var/folders/ys/qjx1ds4s4jqb4sd405dw32j40000gn/T/pip-build-vjynqabb/neupy/neupy/__init__.py", line 4, in <module>
        import theano
    ModuleNotFoundError: No module named 'theano'
```

If I install theano (whatever it is) manually first, it works.

P.S. I'm using Python 3.6.2 if it matters.

**Comment by itdxer** [comment]
Thank you for reporting, I will try to reproduce it

**Comment by itdxer** [comment]
I see, that's definitely a bug, I will try to fix it and post new version. Thanks again for the bug report!

**Comment by itdxer** [comment]
I fixed issue and posted new version 0.6.2

**Comment by dtitov** [comment]
Great, thanks!
